libraries/lib-exceptions/AudacityException.h

```diff
@@ -214,7 +214,7 @@ R GuardedCall(
          // At this point, e is the "current" exception, but not "uncaught"
          // unless it was rethrown by handler.  handler might also throw some
          // other exception object.
-         if (!std::uncaught_exception()) {
+         if (std::uncaught_exceptions() == 0) {
```

Comment:
Are you sure about this `== 0` here? wouldn't `!std::uncaught_exceptions()` be the normal way of doing this (as was the case before)? (same for the ImportAUP further down)

Reply:
`std::uncaught_exception()` returned a boolean value that was true if there were exceptions whereas `std::uncaught_exceptions()` returns the number of exceptions, so to be proper with the types it needs to be compared against an integer. The prior call was checking for when no exceptions were pending (it wanted `std::uncaught_exception` to return false), so that will be when `std::uncaught_exceptions` returns 0.

Reply:
I see.